__tests__/app.test.js

```diff
@@ -42,7 +42,7 @@ describe("/api/articles/:article_id",()=>{
         .get("/api/articles/1")
         .expect(200)
         .then(({body})=>{
-           
+           expect(body.article).toHaveProperty("comment_count")
```

Comment:
This tells me that there is a comment_count, but not that the comment_count is correct.